packages/ui/src/downloader.ts

```diff
@@ -0,0 +1,6 @@
+export function downloader(content: string, name: string): void {
```

Comment:
this is relevant to local content right? if so the name should reflect it. something like `local-content-downloader.ts`